src/main/java/uk/gov/hmcts/reform/cpo/validators/ValidationError.java

```diff
@@ -7,17 +7,20 @@
 
     public static final String ACTION_REQUIRED = "Action is required.";
     public static final String CASE_ID_INVALID = "Case ID has to be a valid 16-digit Luhn number.";
+    public static final String CASE_ID_INVALID_LENGTH = "Case ID has to be 16-digits long.";
     public static final String CASE_ID_REQUIRED = "Case ID is required.";
     public static final String EFFECTIVE_FROM_REQUIRED = "Effective From is required.";
     public static final String ID_INVALID = "ID is invalid.";
     public static final String ID_REQUIRED = "ID is required.";
     public static final String ORDER_REFERENCE_REQUIRED = "Order Reference Party is required.";
     public static final String RESPONSIBLE_PARTY_REQUIRED = "Responsible Party is required.";
+    public static final String ORDER_REFERENCE_INVALID = "Order reference has invalid form";
```

Comment:
```suggestion
    public static final String ORDER_REFERENCE_INVALID = "Order reference has invalid format.";
```
Think this got cut off short 😄